debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -768,6 +769,26 @@ public void shouldTakeBlacklistFiltersIntoAccount() throws Exception {
         assertNoRecordsToConsume();
     }
 
+    @Test
+    public void shouldTakeColumnWhitelistFilterIntoAccount() throws Exception {
+        String setupStmt = SETUP_TABLES_STMT +
+                "CREATE TABLE s1.b (pk SERIAL, aa integer, bb integer, PRIMARY KEY(pk));" +
```

Comment:
Seems like this table s1.b is not used later on?

Reply:
Removed